test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -39,14 +39,15 @@
  * 
  * 1) recompile the JVM with J9VM_OPT_VALHALLA_VALUE_TYPES flag turned on in j9cfg.h.ftl (or j9cfg.h.in when cmake is turned on)
  * 2) cd [openj9-openjdk-dir]/openj9/test/TestConfig
- * 3) export JAVA_BIN=[openj9-openjdk-dir]/build/linux-x86_64-normal-server-release/images/jdk/bin
- * 4) export PATH=$JAVA_BIN:$PATH
- * 5) export JDK_VERSION=Valhalla
- * 6) export SPEC=linux_x86-64_cmprssptrs
- * 7) export BUILD_LIST=functional/Valhalla
- * 8) export AUTO_DETECT=off
- * 9) export JDK_IMPL=openj9
- * 10) make -f run_configure.mk && make compile && make _sanity
+ * 3) export TEST_JDK_HOME=[openj9-openjdk-dir]/build/linux-x86_64-server-release/jdk
```

Comment:
this should point to the `[openj9-openjdk-dir]/build/linux-x86_64-server-release/images/jdk` directory